distribution/docs/src/main/resources/_contents/_draft-registry-contents/managing-registry-contents.adoc

```diff
@@ -0,0 +1,363 @@
+
+${ddf-registry} contains the base registry components, plugins, sources and interfaces needed for ${branding} to function as a registry connecting multiple nodes.
+
+=== ${ddf-registry} Prerequisites
+
+To use the ${ddf-registry}, the following apps/features must be installed:
+
+* <<_managing_ddf_spatial,${ddf-spatial}>>
```

Comment:
The registry requires the Platform App, Security Services App and Admin App in addition to the Spatial App.